go.mod

```diff
@@ -189,3 +189,5 @@ require (
 // kvproto at the same time. You can run `go mod tidy` to make it replaced with go-mod style specification.
 // After the PR to kvproto is merged, remember to comment this out and run `go mod tidy`.
 // replace github.com/pingcap/kvproto => github.com/$YourPrivateRepo $YourPrivateBranch
+
+replace github.com/pingcap/kvproto => github.com/rleungx/kvproto v0.0.0-20230109095105-0fb3bc985ef7
```

Comment:
Is there a related PR?

Reply:
I just sent a PR, see https://github.com/pingcap/kvproto/pull/1034